docs/lib/datastore/fragments/flutter/sync/10-installPlugin.md

```diff
@@ -1,28 +1,3 @@
 ### Add the API plugin
```

Comment:
Do you want to leave this header here? Or add it back later?

Reply:
yeah I was actually thinking about removing the whole section since it's really a no-op for users in developer preview. But then I thought to leave this here to be explicit that we only add the API plugin implicitly in developer preview. What do you recommend, should we remove the whole thing or keep this or keep this without the header?

Reply:
If we leave it, maybe we should make the future call to action more explicit, like:

> “We are currently building the API (GraphQL) category for Flutter. When it becomes available, you will need to add it explicitly in order to use remote data synchronization. This step is not needed in the Developer Preview.”

But I think either approach will work just fine.

Reply:
I like it, stealing.